playground/css/vite.config.js

```diff
@@ -59,6 +59,9 @@ export default defineConfig({
       // },
     },
     preprocessorOptions: {
+      less: {
+        maxWorkers: true,
+      },
```

Comment:
This should use `preprocessorMaxWorkers`

Reply:
Good catch!